### ISSUE TITLE
Please provide the scripts to build and convert indexes on a custom collection

### ISSUE THREAD

**Issue by bianzheng123** [CosimoRulli/emvb#2]

**Comment by CosimoRulli** [comment]
Hey @bianzheng123, 

thank you for your interest. I have added this [script](https://github.com/CosimoRulli/emvb/blob/master/ConvertFaissIndex.ipynb), let me know if you need any further help!

**Comment by bianzheng123** [comment]
Hi, the EMVB method has a lot of parameters for tuning, and the paper shows a little about the parameter configuration. Could you please show the parameters used in the experiment?

**Comment by CosimoRulli** [comment]
Hey @bianzheng123, 

At building time, the number of centroids is $2^{18}$. We tested different values of $M$, as reported in Tables 1 and 2. The number of bits per subspace in PQ (`nbits` in the notebook) is always 8. 

The bash scripts ```results_lotte.sh``` and ```results_msmarco.sh```contain the best configuration we discovered for the querying parameters. As they depend on the dataset you are using, I recommend running a grid search once you have built the index.
